hydra-node/json-schemas/api.yaml

```diff
@@ -1098,19 +1099,7 @@ components:
           properties:
             tag:
               type: string
-              enum: ["NotEnoughFuel"]
-        - title: NoFuelUTXOFound
-          description: |
-            Raised if the internal wallet could not find a fuel output. See user
-            manual how to fuel up the internal wallet.
-          type: object
-          additionalProperties: false
-          required:
-          - tag
-          properties:
-            tag:
-              type: string
-              enum: ["NoFuelUTXOFound"]
```

Comment:
Should: keep the same errors as they still would exist, but only the "fuel marking" is different now. i.e. we just interpret everything held by the internal wallet as fuel

Reply:
https://github.com/input-output-hk/hydra/commit/3cd5abf930e8562d17f28f77b70075e18fa1ecfc